### PR TITLE
Graafinen liittyma

### PR DESCRIPTION
**Pull request by Cl0wnFiesta**
This pull will do:
- Create a main view näkymä view to main view.
- JavaFX import settings so we can use JavaFX imports in imports

### CODE REVIEW

**Review by rasmushy — COMMENTED** [review]
nice

**Review by Henriui — COMMENTED** [review]
Lets test this first with other members before merge

File: GraafinenLiittyma/src/main/resources/com/example/primary.fxml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Henriui) [inline GraafinenLiittyma/src/main/resources/com/example/primary.fxml:1]: Is this file needed?

File: GraafinenLiittyma/src/main/resources/com/example/secondary.fxml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Henriui) [inline GraafinenLiittyma/src/main/resources/com/example/secondary.fxml:1]: Is this file needed?

File: GraafinenLiittyma/src/main/java/com/example/SecondaryController.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package com.example;
```
Comment (Henriui) [inline GraafinenLiittyma/src/main/java/com/example/SecondaryController.java:1]: is this file needed?

File: GraafinenLiittyma/src/main/java/com/example/PrimaryController.java [hunk 4]
```diff
@@ -0,0 +1,12 @@
+package com.example;
```
Comment (Henriui) [inline GraafinenLiittyma/src/main/java/com/example/PrimaryController.java:1]: is this file needed?

**Review by lassib — APPROVED** [review]
Looking good
